src/ResourceManager/SignalR/Commands.SignalR/help/Get-AzureRmSignalR.md

```diff
@@ -84,7 +85,7 @@ The default resource group can be set by `Set-AzureRmDefault -ResourceGroupName
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.IAzureContextContainer
```

Comment:
@LGDoor please regenerate the help to include the fully qualified name for types. You can do this by using the `-UseFullTypeName` parameter with the `Update-MarkdownHelpModule` cmdlet in `platyPS`. More information can be found [here](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module).